fuel/src/main/kotlin/com/github/kittinunf/fuel/core/FuelManager.kt

```diff
@@ -17,7 +17,8 @@ import javax.net.ssl.SSLSocketFactory
 import javax.net.ssl.TrustManagerFactory
 
 class FuelManager {
-    var client: Client by readWriteLazy { HttpClient(proxy) }
+    var client: Client by readWriteLazy { HttpClient(proxy, stethoHook) }
+    var stethoHook: HttpClient.StethoHook? = null
```

Comment:
It seems that the stethoHook usage is isolated to `HttpClient` and can be injected after it has been created. I don't think we need this field at all.